packages/contracts-bedrock/test/L2/L2ToL2CrossDomainMessenger.t.sol

```diff
@@ -289,46 +271,39 @@ contract L2ToL2CrossDomainMessengerTest is Test {
         // contract has a non-zero balance. Thus, we set this contract's balance to zero and we hoax afterwards.
         vm.deal(address(this), 0);
 
-        // Mock the CrossL2Inbox origin to return the L2ToL2CrossDomainMessenger contract
-        vm.mockCall({
-            callee: Predeploys.CROSS_L2_INBOX,
-            data: abi.encodeWithSelector(CrossL2Inbox.origin.selector),
-            returnData: abi.encode(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER)
-        });
-
         // Set the target and message for the reentrant call
         address target = address(this);
         bytes memory message = abi.encodeWithSelector(this.mockTarget.selector, _source, _sender);
 
+        bytes32 msgHash = keccak256(abi.encode(block.chainid, _source, _nonce, _sender, target, message));
+
         // Look for correct emitted event
         vm.expectEmit(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER);
-        emit L2ToL2CrossDomainMessenger.RelayedMessage(
-            keccak256(abi.encode(block.chainid, _source, _nonce, _sender, target, message))
-        );
+        emit L2ToL2CrossDomainMessenger.RelayedMessage(_source, _nonce, msgHash);
 
         // Ensure the target contract is called with the correct parameters
         vm.expectCall({ callee: target, msgValue: _value, data: message });
 
-        // Ensure caller is CrossL2Inbox to prevent a revert from the caller check and that it has sufficient value
-        hoax(Predeploys.CROSS_L2_INBOX, _value);
-
-        // Call the relayMessage function
-        l2ToL2CrossDomainMessenger.relayMessage{ value: _value }({
-            _destination: block.chainid, // ensure the destination is the chain of L2ToL2CrossDomainMessenger
-            _source: _source,
-            _nonce: _nonce,
-            _sender: _sender,
-            _target: target,
-            _message: message
+        // Construct and relay the message
+        ICrossL2Inbox.Identifier memory id =
+            ICrossL2Inbox.Identifier(Predeploys.L2_TO_L2_CROSS_DOMAIN_MESSENGER, 1, 1, 1, _source);
```

Comment:
NIT: the `id` could also be fuzzed and only be updated to avoid using those `1, 1, 1` for the values we are not interested in on the test

Reply:
The same change can be applied in other places where this behavior is repeated :)